arrayfire/array.py

```diff
@@ -1185,20 +1185,35 @@ def to_list(self, row_major=False):
         ct_array, shape = self.to_ctype(row_major, True)
         return _ctype_to_lists(ct_array, len(shape) - 1, shape)
 
-    def __repr__(self):
+    def to_string(self):
```

Comment:
Might works, but don't forget, there are more popular Python IDEs in the market. I've just started to use Visual Studio Code, which is very well designed environment. I can check later if it call __str__ internally or doesn't. 

However I can still see no purpose of printing large arrays. Could you tell me a use case, please?